pyseir/cdc/output_mapper.py

```diff
@@ -0,0 +1,526 @@
+        - value: float
+          Value of target measure at given quantile and forecast date for
+          given location.
+        - location_name: str
+          Name of the state.
+
+
+    Parameters
+    ----------
+    fips: str
+        State or County FIPS code
+    N_samples: int
+        Number SEIR model parameter sets of sample.
+    targets: list(str)
+        Names of the targets to forecast, should be interpretable by Target.
+    forecast_date: datetime.date
+        Date when the forecast is done, default the same day when the mapper
+        runs.
+    next_epi_week: epiweeks.week
+        The coming epi weeks, with the start date of which the forecast time
+        window begins.
+    forecast_time_units: list(str)
+        Time units of the forecast target, should be interpretable by
+        ForecastTimeUnit, currently the mapper only supports unit 'day'.
+    quantiles: list(float)
+        Values between 0-1, which are the quantiles of the forecast target
+        to collect. For default value check QUANTILES.
+    forecast_uncertainty: str
+        Determines how forecast uncertainty is adjusted based on number of
+        days the forecast is made ahead and total days of observations.
+        Should be interpretable by ForecastUncertainty. Currently supports:
+        - 'default': no adjustment
+        - 'naive': rescale the standard deviation by factor (1 + days_ahead
+                   ** 0.5)
+    """
+    def __init__(self,
+                 fips,
+                 N_samples=5000,
+                 targets=TARGETS,
+                 forecast_date=FORECAST_DATE,
+                 next_epi_week=NEXT_EPI_WEEK,
+                 forecast_time_units=FORECAST_TIME_UNITS,
+                 quantiles=QUANTILES,
+                 forecast_uncertainty='default'):
+
+        self.fips = fips
+        self.N_samples = N_samples
+        self.targets = [Target(t) for t in targets]
+        self.forecast_time_units = [ForecastTimeUnit(u) for u in forecast_time_units]
+        self.forecast_date=forecast_date
+        self.forecast_time_range = [datetime.fromordinal(next_epi_week.startdate().toordinal()) + timedelta(n)
+                                    for n in range(FORECAST_WEEKS_NUM * 7)]
+
+        self.quantiles = quantiles
+        self.forecast_uncertainty = ForecastUncertainty(forecast_uncertainty)
+
+        self.model = load_mle_model(self.fips)
+
+        fit_results = load_inference_result(self.fips)
+        forecast_days_since_ref_date = [(t - REF_DATE).days for t in self.forecast_time_range]
+        self.forecast_given_time_range = \
+            lambda forecast, t_list: np.interp(forecast_days_since_ref_date,
+                                               [fit_results['t0'] + t for t in t_list],
+                                               forecast)
+
+        self.result = None
+
+    def run_model_ensemble(self, override_param_names=['R0', 'I_initial', 'E_initial', 'suppression_policy']):
```

Comment:
You can get into trouble if you specify mutable default arguments like lists (e.g., see [this](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)). I would either make this a tuple or define the method as:

```python
def run_model_ensemble(self, override_param_names=None):
    override_param_names = ['R0', 'I_initial', 'E_initial', 'suppression_policy']
    ...
```

Reply:
Ok, tuple sounds better to me, thanks.